### DISCUSSION TITLE
Microphone

### DISCUSSION THREAD

**Discussion by pdifeo** [rustdesk/rustdesk#9219] [Q&A]
Is it possible to redirect the caller's microphone to the remote computer ?

It would be useful for smartworking people who could use audio calls made on the computer in the office.

Thanks

**Comment by pdifeo** [comment]
I return to the argument. I would like as a future implementation what Splashtop does as explained here

https://support-splashtopbusiness.splashtop.com/hc/en-us/articles/360055835272-Microphone-Passthrough

I think it is a very useful feature for smartworkers

**Reply by lixiaotaowx** [reply]
也非常期待这个功能，有了它就能远程协助 而不是远程控制
